### PR TITLE
Add basic docs to readme

### PR DESCRIPTION
**Pull request by CCInc**
## What does this PR do?

Fixes #\<issue_number>

## Before submitting

- [ ] Did you make sure **title is self-explanatory** and **the description concisely explains the PR**?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **test your PR locally** with `pytest` command?
- [ ] Did you **run pre-commit hooks** with `pre-commit run -a` command?

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,9 @@
 
 </div>
 
-**A versatile toolbox for 3D machine learning based on Pytorch Lightning and Hydra.**
+**A versatile framework for 3D machine learning built on Pytorch Lightning and Hydra.**
+
+Please see the [Project Guidelines](#project-guidelines) and [Contributions](#%E2%9D%A4%EF%B8%8F-contributions) sections before diving into the code!
```
Comment (leo-stan) [inline README.md:14]: The link to contribution isn't working

File: README.md [hunk 2]
```diff
@@ -378,6 +380,42 @@ ValueError: Specify tags before launching a multirun!
 
 <br>
 
+## Project Guidelines
```
Comment (leo-stan) [inline README.md:383]: Very cool! I find it easy to understand, well done. Should we add something about the task and how that links models and dataset?

File: README.md (hunk 1 above)
Comment (CCInc) [reply README.md:14]: It was working before precommit came along...

File: README.md (hunk 2 above)
Comment (CCInc) [reply README.md:383]: I think what I should do is add documentation about the entire config code and how it interlinks. I'll do that in the future :smile_cat:

**Pull request merged**